$OpenBSD: patch-src_obconf_glade,v 1.1 2011/10/30 15:32:11 jasper Exp $

don't use a non-zero page size for some spinners (Fix bug #4540)
from upstream git b0465895c15c544111f0dbbcbffab56b476a8749

--- src/obconf.glade.orig	Sun Jan 27 21:30:56 2008
+++ src/obconf.glade	Sun Oct 30 16:23:01 2011
@@ -1743,7 +1743,7 @@ Omnipresent (On all desktops)</property>
 				  <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				  <property name="snap_to_ticks">False</property>
 				  <property name="wrap">False</property>
-				  <property name="adjustment">1 1 100 1 10 10</property>
+				  <property name="adjustment">1 1 100 1 10 0</property>
 				  <signal name="value_changed" handler="on_drag_threshold_value_changed" after="yes" last_modification_time="Thu, 31 May 2007 16:02:00 GMT"/>
 				</widget>
 				<packing>
@@ -1827,7 +1827,7 @@ Omnipresent (On all desktops)</property>
 				  <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				  <property name="snap_to_ticks">False</property>
 				  <property name="wrap">False</property>
-				  <property name="adjustment">20 0 100 1 10 10</property>
+				  <property name="adjustment">20 0 100 1 10 0</property>
 				  <signal name="value_changed" handler="on_resist_window_value_changed" after="yes" last_modification_time="Tue, 23 Sep 2003 22:22:53 GMT"/>
 				</widget>
 				<packing>
@@ -1911,7 +1911,7 @@ Omnipresent (On all desktops)</property>
 				  <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				  <property name="snap_to_ticks">False</property>
 				  <property name="wrap">False</property>
-				  <property name="adjustment">1 0 100 1 10 10</property>
+				  <property name="adjustment">1 0 100 1 10 0</property>
 				  <signal name="value_changed" handler="on_resist_edge_value_changed" after="yes" last_modification_time="Tue, 23 Sep 2003 22:23:04 GMT"/>
 				</widget>
 				<packing>
@@ -2046,7 +2046,7 @@ Omnipresent (On all desktops)</property>
 				      <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				      <property name="snap_to_ticks">False</property>
 				      <property name="wrap">False</property>
-				      <property name="adjustment">0 100 10000 10 10 10</property>
+				      <property name="adjustment">0 100 10000 10 10 0</property>
 				      <signal name="value_changed" handler="on_warp_edge_time_value_changed" after="yes" last_modification_time="Sat, 04 Aug 2007 15:51:02 GMT"/>
 				    </widget>
 				    <packing>
@@ -2488,7 +2488,7 @@ Omnipresent (On all desktops)</property>
 						  <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 						  <property name="snap_to_ticks">False</property>
 						  <property name="wrap">False</property>
-						  <property name="adjustment">1 0 1000 1 10 10</property>
+						  <property name="adjustment">1 0 1000 1 10 0</property>
 						  <signal name="value_changed" handler="on_fixed_x_pos_value_changed" after="yes" last_modification_time="Sat, 12 Jan 2008 18:21:45 GMT"/>
 						</widget>
 						<packing>
@@ -2630,7 +2630,7 @@ Omnipresent (On all desktops)</property>
 						  <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 						  <property name="snap_to_ticks">False</property>
 						  <property name="wrap">False</property>
-						  <property name="adjustment">1 0 1000 1 10 10</property>
+						  <property name="adjustment">1 0 1000 1 10 0</property>
 						  <signal name="value_changed" handler="on_fixed_y_pos_value_changed" after="yes" last_modification_time="Sat, 12 Jan 2008 18:22:16 GMT"/>
 						</widget>
 						<packing>
@@ -2984,7 +2984,7 @@ Omnipresent (On all desktops)</property>
 					  <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 					  <property name="snap_to_ticks">False</property>
 					  <property name="wrap">False</property>
-					  <property name="adjustment">0 0 10000 10 10 10</property>
+					  <property name="adjustment">0 0 10000 10 10 0</property>
 					  <signal name="value_changed" handler="on_focus_delay_value_changed" after="yes" last_modification_time="Sun, 14 Sep 2003 11:24:28 GMT"/>
 					</widget>
 					<packing>
@@ -3246,7 +3246,7 @@ Omnipresent (On all desktops)</property>
 				  <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				  <property name="snap_to_ticks">False</property>
 				  <property name="wrap">False</property>
-				  <property name="adjustment">0 0 10000 10 10 10</property>
+				  <property name="adjustment">0 0 10000 10 10 0</property>
 				  <signal name="value_changed" handler="on_doubleclick_time_value_changed" after="yes" last_modification_time="Thu, 31 May 2007 16:09:40 GMT"/>
 				</widget>
 				<packing>
@@ -3552,7 +3552,7 @@ Omnipresent (On all desktops)</property>
 				  <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				  <property name="snap_to_ticks">False</property>
 				  <property name="wrap">False</property>
-				  <property name="adjustment">0 1 10000 10 10 10</property>
+				  <property name="adjustment">0 1 10000 10 10 0</property>
 				  <signal name="value_changed" handler="on_desktop_popup_time_value_changed" after="yes" last_modification_time="Sat, 04 Aug 2007 16:03:27 GMT"/>
 				</widget>
 				<packing>
@@ -3643,7 +3643,7 @@ Omnipresent (On all desktops)</property>
 			      <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 			      <property name="snap_to_ticks">False</property>
 			      <property name="wrap">False</property>
-			      <property name="adjustment">1 1 100 1 10 10</property>
+			      <property name="adjustment">1 1 100 1 10 0</property>
 			      <signal name="value_changed" handler="on_desktop_num_value_changed" after="yes" last_modification_time="Mon, 15 Sep 2003 01:16:20 GMT"/>
 			    </widget>
 			    <packing>
@@ -3988,7 +3988,7 @@ Omnipresent (On all desktops)</property>
 					  <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 					  <property name="snap_to_ticks">False</property>
 					  <property name="wrap">False</property>
-					  <property name="adjustment">20 0 100 1 10 10</property>
+					  <property name="adjustment">20 0 100 1 10 0</property>
 					  <signal name="value_changed" handler="on_margins_top_value_changed" after="yes" last_modification_time="Thu, 19 Jul 2007 23:50:09 GMT"/>
 					</widget>
 					<packing>
@@ -4230,7 +4230,7 @@ Omnipresent (On all desktops)</property>
 				      <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				      <property name="snap_to_ticks">False</property>
 				      <property name="wrap">False</property>
-				      <property name="adjustment">20 0 100 1 10 10</property>
+				      <property name="adjustment">20 0 100 1 10 0</property>
 				      <signal name="value_changed" handler="on_margins_left_value_changed" after="yes" last_modification_time="Thu, 19 Jul 2007 23:50:31 GMT"/>
 				    </widget>
 				    <packing>
@@ -4443,7 +4443,7 @@ Omnipresent (On all desktops)</property>
 				      <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				      <property name="snap_to_ticks">False</property>
 				      <property name="wrap">False</property>
-				      <property name="adjustment">20 0 100 1 10 10</property>
+				      <property name="adjustment">20 0 100 1 10 0</property>
 				      <signal name="value_changed" handler="on_margins_right_value_changed" after="yes" last_modification_time="Thu, 19 Jul 2007 23:50:41 GMT"/>
 				    </widget>
 				    <packing>
@@ -4678,7 +4678,7 @@ Omnipresent (On all desktops)</property>
 				      <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				      <property name="snap_to_ticks">False</property>
 				      <property name="wrap">False</property>
-				      <property name="adjustment">20 0 100 1 10 10</property>
+				      <property name="adjustment">20 0 100 1 10 0</property>
 				      <signal name="value_changed" handler="on_margins_bottom_value_changed" after="yes" last_modification_time="Thu, 19 Jul 2007 23:50:59 GMT"/>
 				    </widget>
 				    <packing>
@@ -5163,7 +5163,7 @@ Omnipresent (On all desktops)</property>
 				      <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				      <property name="snap_to_ticks">False</property>
 				      <property name="wrap">False</property>
-				      <property name="adjustment">1 0 10000 1 10 10</property>
+				      <property name="adjustment">1 0 10000 1 10 0</property>
 				      <signal name="value_changed" handler="on_dock_float_x_value_changed" after="yes" last_modification_time="Sun, 14 Sep 2003 11:07:38 GMT"/>
 				    </widget>
 				    <packing>
@@ -5208,7 +5208,7 @@ Omnipresent (On all desktops)</property>
 				      <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				      <property name="snap_to_ticks">False</property>
 				      <property name="wrap">False</property>
-				      <property name="adjustment">1 0 10000 1 10 10</property>
+				      <property name="adjustment">1 0 10000 1 10 0</property>
 				      <signal name="value_changed" handler="on_dock_float_y_value_changed" after="yes" last_modification_time="Sun, 14 Sep 2003 11:08:09 GMT"/>
 				    </widget>
 				    <packing>
@@ -5698,7 +5698,7 @@ Omnipresent (On all desktops)</property>
 				      <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				      <property name="snap_to_ticks">False</property>
 				      <property name="wrap">False</property>
-				      <property name="adjustment">300 0 10000 10 10 10</property>
+				      <property name="adjustment">300 0 10000 10 10 0</property>
 				      <signal name="value_changed" handler="on_dock_hide_delay_value_changed" after="yes" last_modification_time="Sun, 14 Sep 2003 11:08:49 GMT"/>
 				    </widget>
 				    <packing>
@@ -5820,7 +5820,7 @@ Omnipresent (On all desktops)</property>
 				      <property name="update_policy">GTK_UPDATE_ALWAYS</property>
 				      <property name="snap_to_ticks">False</property>
 				      <property name="wrap">False</property>
-				      <property name="adjustment">300 0 10000 10 10 10</property>
+				      <property name="adjustment">300 0 10000 10 10 0</property>
 				      <signal name="value_changed" handler="on_dock_show_delay_value_changed" after="yes" last_modification_time="Thu, 31 May 2007 15:17:16 GMT"/>
 				    </widget>
 				    <packing>
